components/identities/identityWarnings.tsx

```
@@ -6,39 +54,52 @@ import Notification from "../UI/notification";
+				This domain has expired. You can renew it by clicking
+				<button>
+					{isExpired
+						? `Expired on ${formattedExpiryDate}`
+						: "Renew Your Domain"}
+				</button>
+			</Notification>
+			<Notification visible={showWarning} severity="error">
+				<>
+					&nbsp;Be careful this domain is not linked to your current
+					address.
+				</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)